packages/server/src/validator/types.ts

```diff
@@ -1,7 +1,32 @@
+/*-
```

Comment:
Let's create a new `types` folder for the `server` package and then host types and interfaces there!